src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/ReportedAdjudicationServiceTest.kt

```diff
@@ -0,0 +1,63 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.services
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.assertj.core.api.Java6Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.gateways.PrisonApiGateway
+import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.gateways.ReportedAdjudication
+import java.time.LocalDateTime
+import javax.persistence.EntityNotFoundException
+
+class ReportedAdjudicationServiceTest {
+  private val prisonApiGateway: PrisonApiGateway = mock()
+  private lateinit var reportedAdjudicationService: ReportedAdjudicationService
```

Comment:
Meh, this can be val.

Reply:
Not sure it can. Also this was taken from `DraftAdjudicationServiceTest`